server/src/schemas/trainingSchema.ts

```diff
@@ -1,5 +1,30 @@
 import { gql } from "apollo-server-express";
 
+export interface TrainingModule {
+  id: number;
+  name: string;
+}
+
+export enum QuestionType {
+  MULTIPLE_CHOICE,
+  CHECKBOXES
+}
+
+export interface Question {
+  id: number | undefined;
+  moduleId?: number;
```

Comment:
Do we need `Question.moduleId`?

Reply:
I needed the moduleID to update the order of all the related questions, I think there's a better way to do this tho